docs/docs/guides/getting_started.md

```diff
@@ -153,7 +113,7 @@ cameras:
         - 0,461,3,0,1919,0,1919,843,1699,492,1344,458,1346,336,973,317,869,375,866,432
 ```
 
-### Step 7: Enable recording (optional)
+### Step 6: Enable recording (optional)
```

Comment:
Might be worth mentioning here that if recordings are left disabled, enabling them in the UI will do nothing